pkg/overrides/limits.go

```diff
@@ -0,0 +1,132 @@
+package overrides
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+	"strings"
+
+	"gopkg.in/yaml.v3"
+)
+
+const (
+	// Error messages
+	ErrInvalidLimits = "the following limits cannot be modified via the overrides API"
+)
+
+// AllowedLimits defines the limits that can be modified via the overrides API
+var AllowedLimits = []string{
+	"max_global_series_per_user",
+	"max_global_series_per_metric",
+	"ingestion_rate",
+	"ingestion_burst_size",
+	"ruler_max_rules_per_rule_group",
+	"ruler_max_rule_groups_per_tenant",
+}
```

Comment:
put this configuration in the runtime config 
https://cortexmetrics.io/docs/configuration/configuration-file/#runtime_configuration_storage_config